# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: mozglue/baseprofiler/public/BaseProfilerMarkers.h
# Commit: e582038fdae2
# Full Hash: e582038fdae25b304ea153cf10f498f78dd3cc71
# Author: Gerald Squelart <gsquelart@mozilla.com>
# Date: 2020-10-17 09:14:56
# Regressor Bug: 1670954
# File Overlap Count: 1
# Description:
#   Bug 1670954 - MarkerThreadId::MainThread() - r=gregtatum
#   
#   Since the main thread is almost always being profiled, this makes it easy to direct important markers there (e.g., FileIO markers from unregistered threads).
#   
#   Tech note: The `#include "BaseProfiler.h"` line was moved to BaseProfilerMarkersPrerequesites.h so that `MarkerThreadId::MainThread()` there can access `profiler_main_thread_id()`.
# ==============================================================================

diff -r a01db300cf22 -r e582038fdae2 mozglue/baseprofiler/public/BaseProfilerMarkers.h
--- a/mozglue/baseprofiler/public/BaseProfilerMarkers.h	Fri Oct 16 22:06:53 2020 +0000
+++ b/mozglue/baseprofiler/public/BaseProfilerMarkers.h	Fri Oct 16 22:07:20 2020 +0000
@@ -33,9 +33,6 @@
 
 #include "mozilla/BaseProfilerMarkersDetail.h"
 
-// TODO: Move common stuff to shared header instead.
-#include "BaseProfiler.h"
-
 #ifndef MOZ_GECKO_PROFILER
 
 #  define BASE_PROFILER_MARKER_UNTYPED(markerName, categoryName, ...)